Settings option on sidebar for class use?

ernbrn opened the pull request:
Hi friends -- I created a link on the sidebar that opens a modal intended for User Settings. 

If it's agreeable to the class, any user setting needs can be put into this modal and they can happily live there. So, if your project has a feature the user can toggle, the UI part can be rendered there.

Code can go in views/user/_settings.

ernbrn committed:
created TagMailer mailer

ernbrn committed:
Edited mailer model and views

*Added html and plain text email templates
*Created a email method to send these emails

cndreisbach commented:
@ernbrn Could you merge the upstream master and repush this branch? Right now, the tests are failing.

Note that I changed something about how tests run, so you'll need to run `brew install phantomjs` locally before running again.

ernbrn committed:
working on tagmailer

ernbrn committed:
working on tagmailer

ernbrn committed:
fixed typo in mailer

ernbrn committed:
does not have author in database ?

ernbrn committed:
Merge remote-tracking branch 'upstream/master' into tag-emails

ernbrn committed:
Merge remote-tracking branch 'upstream/master' into tag-emails

ernbrn committed:
Merge remote-tracking branch 'upstream/master' into tag-emails

ernbrn committed:
merging upstream

ernbrn committed:
deleted _settings.html.erb

ernbrn committed:
working on simple_form_for

ernbrn committed:
Email updates are completely functional

TODO: Write tests, make functionality so that when an old book is updated with an old tag,
the user receives an email about it, but not a duplicate one about a previous tag.

ernbrn committed:
rectifying merge conflicts

ernbrn committed:
cannot add tags after merging upstream

ernbrn commented:
My email updates are fully functional (!) and users can opt in or out of emails (default is out) at any time. However, when I merged the master branch in with mine, I cannot seem to create any tags -- some bug somewhere.
